src/apify_client/clients/base/actor_job_base_client.py

```diff
@@ -73,6 +73,13 @@ def _abort(self, gracefully: Optional[bool] = None) -> Dict:
         )
         return parse_date_fields(_pluck_data(response.json()))
 
+    def _reboot(self) -> Dict:
```

Comment:
I would not put `_reboot` in this class since reboot is only for Actor Runs and not builds. (it's basically a specific version of metamorph, so I'd take inspiration from that method)